.github/workflows/BuildDockerContainer.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
         uses: docker/setup-qemu-action@v3
         with:
           platforms: arm64
+          build-args: PARALLEL_MAKE_ARG=-j4
```

Comment:
Delete this line but add the line twice in DemoContainer.yaml (for the `build-push-action`). Squash.

Reply:
Done! Thanks!